### PR TITLE
Feature/form banner support

### PR DESCRIPTION
**Pull request by 2piJareem**
## Checklist
- [x] I'm using the correct PHP Version (7.4 for current, 7.2 for legacy).
- [x] I've added comments to any new methods I've created or where else relevant.
- [x] I've replaced magic method usage on DbService classes with the getInstance() static method.
- [x] I've written any documentation for new features or where else relevant in the docs [repo](https://github.com/2pisoftware/cmfive-docs).

## Description

Provide support for Text Banners and Lines within bootstrap5 forms
Enable the choosing of Groups within the Insight module.
Manage Groups within the Admin module so that Groups and Users are sorted alphabetically ignoring case when viewing or selecting from a list.

## Changelog

**Added support for a text comment/banner to be shown in a bootstrap5 form.**
system/classes/html/form/Banner.php

**Added support for horizontal lines in a bootstrap5 form**
system/classes/html/form/Line.php

**Manage Groups within Insight reports**
system/modules/insights/actions/index.php
system/modules/insights/actions/manageMembers.php
system/modules/insights/actions/members/editMembers.php

**Spelling fix in button Title**
system/modules/insights/actions/members/deleteMembers.php

**Sorting when viewing Users/Groups**
system/modules/admin/actions/groups.php
system/modules/admin/actions/groupmember.php
system/modules/admin/actions/moreInfo.php

refs:
issues:

## Other Information

NB cmfive-core feature/InsightsReportingBootstrap5 will need to be merged with this branch to get the "Adding of Groups to Insight Reports" functionality.

Other branches depend upon this branch:
bridge_cmfive_module feature/Phase3
facilitator_bridge_module feature/Phase3

Docs pull request:

### CODE REVIEW

**Comment by 2piJareem** [comment]
Changed PR to merge with develop

**Comment by adam-buckley** [comment]
@dragonflyfree can you check this branch and maybe reconcile it against the admin module stuff you've done?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -61,19 +62,19 @@ jobs:
...
       # Define cache for .composer
-      - name: Cache .composer
-        uses: actions/cache@v4
-        id: cache
-        with:
-          path: |
-            /tmp/.composer
-            ${{ github.workspace }}/boilerplate/composer
-          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+      # - name: Cache .composer
```
Comment (mattbell87) [inline .github/workflows/ci.yml:65]: I reckon get rid of this and do it from scratch again later when we want to tackle it

File: system/modules/insights/actions/index.php [hunk 2]
```diff
@@ -22,34 +22,40 @@ function index_ALL(Web $w)
...
+                            $userHasAccess = AuthService::getInstance($w)->isUserGroupMemberRecursive($member->user_id, $user_id);  // $member->user_id may be a user or a group
                             if ($userHasAccess) {
                                 break;
-                            };
+                            }
                         }
                     }
                     if ($userHasAccess) {
-                        $row = [];
+                        // $row = [];
```
Comment (mattbell87) [inline system/modules/insights/actions/index.php:40]: this will def always be an array?

**Review by mattbell87 — COMMENTED** [review]
Otherwise looks good

File: .github/workflows/ci.yml (hunk 1 above)
Comment (adam-buckley) [reply .github/workflows/ci.yml:65]: I left it in for reference but I can get rid of it

**Review by mattbell87 — APPROVED** [review]

**Pull request merged**